src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
We have fixed similar problem in a couple places using the attribute - check NDP\clr\src\debug\daccess\daccess.cpp